pilot/pkg/model/context.go

```diff
@@ -492,6 +492,18 @@ func compareVersion(ov, nv int) int {
 
 var NodeTypes = [...]NodeType{SidecarProxy, Router, Waypoint, Ztunnel}
 
+func (node *Proxy) FirstAddressOrNil() string {
+	if node == nil {
+		return ""
+	}
+
+	if len(node.IPAddresses) == 0 {
+		return ""
+	}
+
+	return node.IPAddresses[0]
+}
```

Comment:
```suggestion
	if node == nil || len(node.IPAddresses) == 0 {
		return ""
        }
	return node.IPAddresses[0]
}
```

Reply:
Thank you, it looks more tidy.
I updated the code :+1: